### PR TITLE
Minor modifications in app/models/

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/models/message/Message.php [hunk 1]
```diff
@@ -123,11 +128,7 @@ public function set($stanza, $parent = false)
                 $this->body = trim(html_entity_decode($this->body));
             }
 
-            if($stanza->markable) {
-                $this->markable = true;
-            } else {
-                $this->markable = false;
-            }
+            $this->markable = $stanza->markable;
```
Comment (edhelas) [inline app/models/message/Message.php:131]: Here it check if the XML tag is there, $stanza->markable is not a boolean

File: app/models/contact/Contact.php [hunk 2]
```diff
@@ -302,7 +302,7 @@ function getAlbum()
         $uri = str_replace(
```
Comment (edhelas) [inline app/models/contact/Contact.php]: You can simply remove the getAlbum() methode, it's not called anywhere anymore

Comment (ghost) [reply app/models/contact/Contact.php]: Ok, I will.

File: app/models/message/Message.php (hunk 1 above)
Comment (ghost) [reply app/models/message/Message.php:131]: What is the type of `$stanza`? I see no method `attributes` for [DOMElement](https://secure.php.net/manual/en/class.domelement.php).

**Comment by edhelas** [comment]
Most of those changes are already in master now :)

**Comment by ghost** [comment]
I rebased if you would like to integrate the rest.

**Pull request closed without merging**